src/Symfony/Component/Console/Style/StyleInterface.php

```diff
@@ -47,6 +47,13 @@ public function listing(array $elements);
     public function text($message);
 
     /**
+     * Formats secondary text of less importance than text().
+     *
+     * @param string|array $message
+     */
+    public function comment($message);
```

Comment:
Isn't it a BC break ? I think the `StyleInterface` is already implemented by some third-party libraries.